src/docs/truffle/overview.md

```diff
@@ -5,7 +5,9 @@ layout: docs.hbs
 <img style="max-width: 160px;" src="/img/truffle-logo-dark.svg" alt="Truffle Logo" />
 
 <div class="text-center docs-badges">
-  [![Join the chat at https://gitter.im/consensys/truffle](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/consensys/truffle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+  <a href="https://gitter.im/consensys/truffle?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge">
```

Comment:
can you actually fix this in the other `overview.md` files while you're at it?

Reply:
Done!